cli/internal/cmdutil/cmdutil.go

```diff
@@ -183,6 +174,21 @@ func (h *Helper) GetCmdBase(cliConfig config.CLIConfigProvider) (*CmdBase, error
 			remoteConfig.TeamID = vercelArtifactsOwner
 		}
 	}
+
+	// Primacy: Arg > Env
+	val, ok := os.LookupEnv("TURBO_REMOTE_CACHE_TIMEOUT")
```

Comment:
at the risk of extra nesting we can prevent duplicate work here by moving it into the else clause for 188